amass/core/config.go

```diff
@@ -342,6 +347,15 @@ func (c *Config) LoadSettings(path string) error {
 			c.AddNumbers = alterations.Key("add_numbers").MustBool(true)
 			c.MinForWordFlip = alterations.Key("minimum_for_word_flip").MustInt(2)
 			c.EditDistance = alterations.Key("edit_distance").MustInt(1)
+			if alterations.HasKey("wordlist_file") {
```

Comment:
Shouldn't this be an option for command line parameters as well? Here's it's only implemented in the config file.